gammapy/image/models/tests/test_core.py

```diff
@@ -42,6 +43,22 @@ def test_sky_disk():
     assert_allclose(val.value, desired)
 
 
+def test_sky_ellipse():
+    model = SkyEllipse(2 * u.deg, 2 * u.deg, 1 * u.deg, 0.8, 30 * u.deg)
+    m_allsky = WcsGeom.create(
+        binsz=0.01, width=(3, 3), skydir=(2, 2), coordsys="GAL", proj="AIT"
+    )
+    coords = m_allsky.get_coord()
+    lon = coords.lon * u.deg
+    lat = coords.lat * u.deg
+    vals = model(lon, lat)
+    assert vals.unit == "sr-1"
+    mymap = Map.from_geom(m_allsky, data=vals.value)
+    assert_allclose(
+        np.sum(mymap.quantity * u.sr ** -1 * m_allsky.solid_angle()), 1, rtol=5.0e-4
+    )
+
```

Comment:
Maybe add a second test case with an even larger radius for the ellipse (~10 deg) to check that the normalisation is correct?

Reply:
Yes sure! I'll do that. Altough I think it may somewhat increase the computation time for the test

Reply:
Of course, you can choose a coarser pixel size to limit the computation time...